docs/data-management/post-session.md

```diff
@@ -374,11 +374,60 @@ ses-01
         └── sub-pilot_ses-01_task-rest_physio.tsv.gz
 ```
 
-
 ### Incorporate into version control with DataLad
 
 !!!info "Initiating the version-controled dataset"
 
-    Once at the beginning of the project, the datalad dataset will be created:
+    Once at the beginning of the project, the DataLad [1] dataset should be created. In case, the preprocessing will be executed on an HPC, check out the foldable admonition for tips on how to install DataLad on the HCP.  
+
+    ??? tip "Installing DataLad with Conda"
+        If datalad is not installed on the HPC, the most convenient user-based installation can be achieved by using Conda.
+
+        !!! tip "Execution within High Performance Computing (HPC)"
+    
+            HPC systems typically recommend using their login nodes only for tasks related to job submission, data management, and preparing job scripts.
+            Therefore, the execution of resource-intensive tasks such as *fMRIPrep* or building containers on login nodes can negatively impact the overall performance and responsiveness of the system for all users.
+            Interactive sessions are a great alternative when available.
+            For example, in the case of systems operating SLURM, the following command would open a new interactive session:
+            ``` bash
+            srun --nodes=1 --ntasks-per-node=1 --time=01:00:00 --pty bash -i
+            ```
```

Comment:
I don't think this particular tip should be in the "Initiating the version-controled dataset" or "Installing DataLad with Conda" admonitions. Maybe it could be moved to the beginning of `mriqc.md` where we will actually start running things (if we expect to run MRIQC on the HPC).

Reply:
@acionca The point I wanted to make is that you should not install DataLad on the login node. So you should open an interactive session using srun before.
It doesn't seem to get across though.
Can you rewrite it to make it clearer?

Reply:
@celprov I added a few suggestions and modification in a [PR](https://github.com/celprov/hcph-sops/pull/1) on your original branch.

Reply:
@acionca Oooh I didn't know you can do that :bulb:  
Thanks a lot!